source/pr_cmds.c

```diff
@@ -1487,330 +1487,286 @@ This is where the magic happens
 =================
 */
 
-int closedset[MAX_WAYPOINTS]; // The set of nodes already evaluated.
-int openset[MAX_WAYPOINTS];//Actual sorted open list
-int opensetRef[MAX_WAYPOINTS];//Reference values of open list
-int opensetLength;//equivalent of javaScript's array[].length;
+
 #define MaxZombies 16
 
+
+#define WAYPOINT_SET_NONE 	0
+#define WAYPOINT_SET_OPEN 	1
+#define WAYPOINT_SET_CLOSED	2
+
+
+char waypoint_set[MAX_WAYPOINTS]; // waypoint_set[i] contains the set identifier for the i-th waypoint
+unsigned short openset_waypoints[MAX_WAYPOINTS]; // List of waypoints currently in the open set sorted by heuristic cost (index 0 contains lowest cost waypoint)
+unsigned short openset_length; // Current length of the open set
 zombie_ai zombie_list[MaxZombies];
-//Debug//
-void printSortedOpenSet()
-{
-	Con_Printf("Sorted!: ");
-	int qr;
-	for(qr = 0; qr < opensetLength; qr++)
-	{
-		Con_Printf("%i, ",(int)waypoints[openset[qr]].f_score);
+
+
+//
+// Debugs prints the current sorted list of waypoints in the open set
+//
+void sv_way_print_sorted_open_set() {
+	Con_Printf("Sorted open-set F-scores: ");
+	for(int i = 0; i < openset_length; i++) {
+		Con_Printf("%.0f, ",waypoints[openset_waypoints[i]].f_score);
 	}
 	Con_Printf("\n");
 }
```

Comment:
Can you make these `Con_Dprintf`s instead?

Reply:
On second thought, I think keeping them as-is makes sense. These are in a debug function that's not currently called. If calls to them are added back in, it's because we're explicitly looking for their output in a debug build, and it'd be nice to not be inundated with other dev logs. (Just like: https://github.com/nzp-team/dquakeplus/blob/8440319b4b21dad8b1865cd25274ea1862eb79f2/source/pr_cmds.c#L2067)